thorlcr/master/thgraphmanager.cpp

```diff
@@ -1039,10 +1048,17 @@ bool CJobManager::executeGraph(IConstWorkUnit &workunit, const char *graphName,
             if (stdDev)
             {
                 Owned<IStatisticGatherer> collector = createGlobalStatisticGatherer(wu);
-                StatsScopeId graphScopeId(graphName);
+                StatsScopeId wfidScopeId(SSTworkflow, wfid);
+                StatsScopeId graphScopeId;
```

Comment:
trivial: Why the change from 
`StatsScopeId graphScopeId(graphName)`

Reply:
not sure, reverted.